sdks/builds/ios.mk

```diff
@@ -149,14 +156,16 @@ ios-targettv_AC_VARS = \
 	ac_cv_func_execvp=no            \
 	ac_cv_func_signal=no
 ios-targetwatch_AC_VARS = $(ios-targettv_AC_VARS)
+ios-targetwatch64_32_AC_VARS = $(ios-targettv_AC_VARS)
 
 # ios-target32_BITCODE_MARKER=-fembed-bitcode-marker
-$(eval $(call iOSDeviceTemplate,target32,arm,armv7))
-$(eval $(call iOSDeviceTemplate,target32s,arm,armv7s))
+$(eval $(call iOSDeviceTemplate,target32,arm-apple-darwin10,armv7))
+$(eval $(call iOSDeviceTemplate,target32s,arm-apple-darwin10,armv7s))
 # ios-target64_BITCODE_MARKER=-fembed-bitcode-marker
-$(eval $(call iOSDeviceTemplate,target64,aarch64,arm64))
-$(eval $(call iOSDeviceTemplate,targettv,aarch64,arm64))
-$(eval $(call iOSDeviceTemplate,targetwatch,armv7k,armv7k))
+$(eval $(call iOSDeviceTemplate,target64,aarch64-apple-darwin10,arm64))
+$(eval $(call iOSDeviceTemplate,targettv,aarch64-apple-darwin10,arm64))
+$(eval $(call iOSDeviceTemplate,targetwatch,armv7k-apple-darwin10,armv7k))
+$(eval $(call iOSDeviceTemplate,targetwatch64_32,aarch64-apple-darwin10_ilp32,arm64_32))
```

Comment:
@luhenry is the `archive` target automatically going to pick up that new target?

Reply:
resolved by https://github.com/mono/mono/pull/12055